FindMKL_ScaLAPACK.cmake

```diff
@@ -0,0 +1,62 @@
+# Uses MKLROOT environment variable or CMake's MKL_ROOT to find MKL ScaLAPACK.
+#
+# Variables:
+#     ScaLAPACK_FOUND
+#
+# Imported Targets: 
+#     MKL::ScaLAPACK
+#
+# Note: The default is MPICH. Do not mix MPI implementations.
+#
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
+include(FindPackageHandleStandardArgs)
+
+find_package(MPI REQUIRED)
+find_package(MKL REQUIRED)
```

Comment:
As already said in the main comment, I suggest to merge it in the main MKL script and enable it as a COMPONENT like in #2.